tests/AgentTests/SQLiteTests.hs

```diff
@@ -71,6 +73,12 @@ storeTests = withStore do
       describe "SndQueue exists" testCreateSndMsg
       describe "SndQueue doesn't exist" testCreateSndMsgNoQueue
 
+testCompiledThreadsafe :: SpecWith SQLiteStore
+testCompiledThreadsafe = do
+  it "should throw error if compiled sqlite library is not threadsafe" $ \store -> do
+    compileOptions <- DB.query_ (dbConn store) "pragma COMPILE_OPTIONS;" :: IO [[T.Text]]
+    shouldNotContain compileOptions [["THREADSAFE=0"]]
```

Comment:
better to write it as ```compileOptions `shouldNotContain` [["THREADSAFE=0"]]```

Reply:
but I don't think it is correct - try testing for what it definitely contains

Reply:
yep, right.. I was trying `compileOptions shouldNotContain [["THREADSAFE=0"]]` but missed backtics. But it is correct, I checked